examples/ipynbtests.sh

```diff
@@ -64,7 +64,7 @@ py.test --nbval --current-env \
     test_snapshot.ipynb \
     test_netcdfplus.ipynb \
     test_cv.ipynb \
-    test_pyemma.ipynb \
+    # test_pyemma.ipynb \ got covered by unittests instead
```

Comment:
you'll need to fully remove this line; the continuation `\` now has a comment after it, so the `||` underneath is no longer part of the same line in bash.